git.md

```diff
@@ -6,11 +6,35 @@ Please see below for curated Git Resources.
 - [The Turing Way][2]: "Version Control" (also includes version control for
 data).
 - [Software Carpentry][3]: "Version Control With Git."
+- [Creating SSH Keys](creating-ssh-keys)
 
 ### Git Training
 We have supplementary questions and answers to go with Pro Git [here][4].
 
+### Creating SSH Keys
+Creating SSH keys allows you to easily authenticate to GitHub in order to
+pull and push changes without having to enter your username and password
+every time.
+You'll need to set up one key pair per machine (e.g., one for your local
+machine, one for biowulf, etc.)
+We recommend following GitHub's instructions [here][ssh-keys] to create SSH
+keys.
+Please create and remember your SSH key password!
+You will only need to enter it ocassionally.
+Make sure that once you've set up your SSH keys, your remotes are set up to
+use the SSH addresses rather than HTTPS.
+You can do this by going to the repository, clicking on the "code" tab, and
+then pressing the "code" drop-down menu.
+Click "SSH" and then the clipboard icon to copy the new link.
```

Comment:
It may be helpful to add that when clicking "code" to select "SSH" Github may mention that no sshkey is associated with the account and offer a [link](https://github.com/settings/ssh/new) to do so.

One can follow the link which prompts them to enter/save the pubkey generated from the steps you outlined above, thus linking said key to the Github account.

This is a necessary step before the following steps can be addressed (git remove remove+add)

Reply:
Good point, thanks @aykhojandi. It occurs to me that for users with multiple machines, the second machine may not have that SSH key and they won't be prompted to fill it, so I'll add instructions for associating the keys.

Reply:
I think 4ef48f1 would fix this.